src/FritzBox/Converter.php

```diff
@@ -171,7 +171,7 @@ private function getPhoneNumbers($card): array
             }
             // get type
             $type = 'other';
-            $telTypes = isset($card->TEL[$key]->parameters['TYPE']) ? strtoupper($card->TEL[$key]->parameters['TYPE']) : '';
+            $telTypes = strtoupper(isset($card->TEL[$key]->parameters['TYPE'])) ?: '';
```

Comment:
Das kann so nicht funktionieren! Das isset muss weg, Klammern nach hinten. Getestet.

Reply:
NO - neither way!
TestCase Okay! But...
Real life test:
Notice: Undefined index: TYPE in C:\Users\Admin\Documents\Development\Git\carddav2fb\src\FritzBox\Converter.php on line 174
Contact without phone numbers will be skipped
Because "Das isset muss weg,"

Reply:
siehe andere kommentare. isset muss weg.

Reply:
Wie ich geschrieben habe: wenn isset() weg ist, dannläuft zwar des TestConverter fehlerfrei durch (weil kein defaultContact definiert ist, der ohne Rufnummer ist!
Im Life-Test führt das zu:

> Notice: Undefined index: TYPE in C:\Users\Admin\Documents\Development\Git\carddav2fb\src\FritzBox\Converter.php